### PR TITLE
FE Style & Router Rough Out

### PR DESCRIPTION
**Pull request by averyfairbanks**
STYLE CHANGES AND SUCH

The PR has a :sunglasses: guy in it. I didn't know that was possible

### CODE REVIEW

File: src/sign-in/SignInButton.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
+.gsi-material-button {
```
Comment (averyfairbanks) [inline src/sign-in/SignInButton.css:1]: This all comes from here: https://developers.google.com/identity/branding-guidelines#render-html-button

If there's an alternative you'd prefer feel free to say so!

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (averyfairbanks) [inline tsconfig.json:1]: Okay... so this and the next tsconfig should handle weirdness around what imports can be called where, what kind of jsx we're using, etc.

Comment (averyfairbanks) [reply tsconfig.json:1]: Please feel free to check me.

File: src/index.css [hunk 3]
```diff
@@ -1,10 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap');
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family:  "Quicksand", sans-serif;
```
Comment (averyfairbanks) [inline src/index.css:5]: TODO: Add this font using chakra's system theme arg and leave the others a a fallback.

File: src/App.tsx [hunk 4]
```diff
@@ -35,22 +37,23 @@ const App = () => {
   }, [auth, user]);
 
   if (isLoading) {
-    return <div className="App">
-      Loading...
-    </div>
+    return (
+      <div className="centered">
+        <Spinner size="lg" color="var(--logo-color)" />
```
Comment (averyfairbanks) [inline src/App.tsx:42]: The spinner color matches the logo :)

**Review by rkaufman13 — APPROVED** [review]

**Pull request merged**